### ISSUE TITLE
_cgo tegola_windows_amd64_cgo binary not built with geopackage support as per documentation

### ISSUE THREAD

**Issue by millionpoundhat** [go-spatial/tegola#465]
tegola_windows_amd64_cgo.exe serve --config=geopackage-tegola.toml

2018-07-19 12:06:29 [INFO] config.go:191: loading local config (geopackage-tegola.toml)
2018/07/19 12:06:29 root.go:74: No providers registered by the name: gpkg, known providers(debug,postgis)

Sanitised TOML 
```
[webserver]
port = ":8181"

#providers geopackage
[[providers]]
name = "geoviz"
type = "gpkg"
filepath = "C:\\Users\\path\\to\\geopackage\\geoviz.qgz"

#provider layers 
[[providers.layers]]
	name = "counts_in_boroughts"
	tablename = "geoviz.counts_in_boroughts"

	[[maps]]
	name = "viz"

 	[[maps.layers]]
	provider_layer = "viz.counts_in_boroughts"
	min_zoom = 0
	max_zoom = 18
```

**Comment by ARolek** [comment]
@gdey did your manual builds of v0.6.x get pulled down? I don't see them anymore but I know you addressed this manually pending completion of #420.

**Comment by ARolek** [comment]
the v0.7.0 release is now available and addresses this issue.
